onnxruntime/python/tools/transformers/benchmark_gpt2.py

```diff
@@ -336,12 +414,16 @@ def main():
         onnx_model_path = os.path.join(output_dir, 'gpt2_past_optimized.onnx')
         m.save_model_to_file(onnx_model_path)
 
-    if args.use_gpu and 'CUDAExecutionProvider' not in onnxruntime.get_available_providers():
-        logger.warning("Please install onnxruntime-gpu package to test GPU inference.")
+    if args.use_gpu and 'CUDAExecutionProvider' not in onnxruntime.get_available_providers(
```

Comment:
Did you run the reformatpython.bat? It seems that line length limit is less than 120 in your format.